pkg/operator/ceohelpers/bootstrap.go

```diff
@@ -182,20 +182,14 @@ func IsBootstrapComplete(configMapClient corev1listers.ConfigMapLister, staticPo
 		return false, nil
 	}
 
-	// now run check to stability of revisions
 	_, status, _, err := staticPodClient.GetStaticPodOperatorState()
 	if err != nil {
 		return false, fmt.Errorf("failed to get static pod operator state: %w", err)
 	}
+
 	if status.LatestAvailableRevision == 0 {
 		return false, nil
 	}
-	for _, curr := range status.NodeStatuses {
-		if curr.CurrentRevision != status.LatestAvailableRevision {
-			klog.V(4).Infof("bootstrap considered incomplete because revision %d is still in progress", status.LatestAvailableRevision)
-			return false, nil
-		}
-	}
 
 	// check if etcd-bootstrap member is still present within the etcd cluster membership
 	membersList, err := etcdClient.MemberList(context.Background())
```

Comment:
alternatively, maybe smarter is to swap the order of the checks.

If we observe the etcd-bootstrap node is not there anymore, bootstrap is completely done and we don't need to check revisions anymore.